### DISCUSSION TITLE
No glitch in graphics with EDID patch

### DISCUSSION THREAD

**Discussion by emrah7celik** [SkyrilHD/Dell-Exx50-Hackintosh#6] [General]
The graphics glitch after cold boot is no more after I enabled HiDPI options through the [one-key-hidpi](https://github.com/xzhih/one-key-hidpi) script with the patch option (Enable HIDPI with EDID). I am not sure why, and it might be entirely coincidental, but if you are experiencing the same issue, give it a try. Not to mention this script improves readability on FHD+ displays by enabling larger text options.

**Comment by SkyrilHD** [comment]
Is this the glitch you mention about? https://github.com/newyb/Something-useful-of-E7250/blob/main/BUG2%20at%20sign.jpg

**Reply by emrah7celik** [reply]
Yes, I got that screen _every time_ I booted until I made the laptop sleep and back on again. After using the aforementioned script, it was all normal!

**Reply by SkyrilHD** [reply]
Thanks for the fix
